pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -130,7 +130,10 @@ export class LogsPanel extends React.Component {
     render() {
         return (
             <div className="panel panel-default cockpit-log-panel" role="table">
-                <div className="panel-heading">{this.props.title}</div>
+                <div className="panel-heading">
+                    <h2 className="panel-title">{this.props.title}</h2>
+                    <button className="link-button" role="link" onClick={e => cockpit.jump(this.props.goto_url)}>{_("Go to all logs")}</button>
```

Comment:
Thinking this doesn't need to be a verb. It's evident from the link styling that it will go to somewhere, so I would probably go for **All logs** instead.

Reply:
Done